shared/src/main/java/com/couchbase/lite/ReplicatedDocument.java

```diff
@@ -58,17 +68,10 @@ public String getID() {
     }
 
     /**
-     * The current deleted status of the document.
-     */
-    public boolean isDeleted() {
-        return isDeleted;
-    }
-
-    /**
-     * The current access removed status of the document.
+     * The current status flag of the document. eg. deleted, access removed
      */
-    public boolean isAccessRemoved()  {
-        return isAccessRemoved;
+    public Set<DocumentFlag> flags() {
```

Comment:
Should it be EnumSet instead of Set (ReplicationFilter interface uses EnumSet)? 

Could you do some research that normally it should be Set or EnumSet?

Reply:
Most EnumSet example usages is EnumSet.